examples/postprocess/bloom.js

```diff
@@ -1,179 +0,0 @@
-(function() {
```

Comment:
If you delete postprocess example it does not make sense to have the new one as new-postprocess does it?